tensorflow/lite/kernels/cast.cc

```diff
@@ -27,6 +27,10 @@ limitations under the License.
 #include "tensorflow/lite/kernels/kernel_util.h"
 #include "tensorflow/lite/kernels/op_macros.h"
 
+#ifdef __ARM_NEON
+  #include <arm_neon.h>
```

Comment:
```suggestion
#include <arm_neon.h>
```

(although it does not matter for the compiler, it follows the style)